site/docs/terms.md

```diff
@@ -33,11 +33,11 @@ Each manifest file in the manifest list is stored with information about its con
 
 A **manifest file** is a metadata file that lists a subset of data files that make up a snapshot.
 
-Each data file in a manifest is stored with a [partition tuple](#partition-tuple), column-level stats, and summary information used to prune splits during [scan planning](../performance#scan-planning).
+Each data file in a manifest is stored with a [partition tuple](#partition-tuple), column-level stats, and summary information used to prune splits during [scan planning](../docs/performance.md#scan-planning).
```

Comment:
This link works for me on iceberg.apache.org. If you're trying to read the docs from github, the links probably won't work. But the this is for the ASF site.

Reply:
ok, I will close the pr